pkgs/top-level/all-packages.nix

```diff
@@ -5867,8 +5867,6 @@ in
 
   soundkonverter = kdeApplications.callPackage ../applications/audio/soundkonverter {};
 
-  souper = callPackage ../development/compilers/souper { };
```

Comment:
Can you put a `souper = throw "<reason for removal>";`? I don't like removing things without any trace.